frontend/src/queries/schema.ts

```diff
@@ -125,9 +125,17 @@ export interface HogQLQueryResponse {
     columns?: any[]
 }
 
+/** Filters object that will be converted to a HogQL {filters} placeholder */
+export interface HogQLFilters {
+    properties?: AnyPropertyFilter[]
+    dateFrom?: string | null
+    dateTo?: string | null
```

Comment:
We're using an additional `dateRange` nesting for `dateFrom` and  `dateTo` for insight queries. This is quite useful, since you usually want to get both these properties at the same time.

In any case I think we should keep the schema consistent across queries and either adapt the HogQLFilters or the insight queries.

Reply:
Additionally, there's a bunch of logic around date ranges that we probably want to keep consistent and not duplicate (e.g. using what dateFrom to use if there isn't one provided, or if "all" is provided)